docs/guides/admin/docs/releasenotes.md

```diff
@@ -71,8 +71,8 @@ configure asset upload options in more detail [[#6362](https://github.com/openca
 - Waveform peaks are now more visible, but that behavior is configurable.
   [[#6028](https://github.com/opencast/opencast/pull/6028)]
 
-For more details, please take a look at the [full changelog](changelog.md). If you want to update Opencast from a
-previous version, you should also read the [upgrade guide](upgrade.md).
+For more details, please take a look at the [full changelog](changelog/opencast{{ opencast_major_version() }}.md).
```

Comment:
`changelog/opencast{{ opencast_major_version() }}.md` resolves to `changelog/opencast17.md` at runtime, which generates `WARNING -  Doc file 'releasenotes.md' contains a link 'changelog/opencast17.md', but the target is not found among documentation files.`

Reply:
Good catch, fixed!